src/apns_connection.erl

```diff
@@ -127,6 +127,10 @@ default_connection(certdata, ConnectionName) ->
    , keydata    => Key
    , timeout    => Timeout
    , type       => certdata
+   % starting with backoff = 0 because 2^0=1, so we start from 1 sec delay,
+   % delays chain by default will be 1 2 4 16 65536 seconds
+   , backoff         => 0
+   , backoff_ceiling => application:get_env(apns, backoff_ceiling, 10)
```

Comment:
Maybe it would be worth setting that option in `.app.src` and then just using `application:fetch_env`?

Reply:
There are also another options supposed to be configured in case of usage default connection configs,  so I think we can just use `get_env/2` here. It will be consistent with other config opts.
But in this case docs should be updated as well to reflect these new options.

Reply:
P.S. Updated the code and README